modin/core/execution/ray/generic/io/io.py

```diff
@@ -134,56 +148,51 @@ def func(df, **kw):
                 Arguments to pass to ``pandas.to_csv(**kw)`` plus an extra argument
                 `partition_idx` serving as chunk index to maintain rows order.
             """
-            if kw["partition_idx"] != 0:
+            idx = kw["partition_idx"]
+            _kwargs = kwargs.copy()
```

Comment:
```suggestion
            csv_kwargs = kwargs.copy()
```
I suggest renaming the variable for clarity.

Also pls add a small comment on why copy.